src/html/alert-banner.html

```diff
@@ -42,7 +43,6 @@
     font-size: var(--fs-micro);
     font-weight: var(--fw-regular);
     padding: var(--spacing-m);
-    text-align: center;
```

Comment:
What does this look like now?

Reply:
![image](https://user-images.githubusercontent.com/29397451/114486886-95850300-9c49-11eb-9b64-969e3aacaa66.png)